<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlotEditorWnd</class>
 <widget class="QDialog" name="SlotEditorWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>551</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListView" name="lstSlots">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>221</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblSlotsList">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Список слотов</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSlotName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSlotType">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblSlotType">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSlotDomain">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Домен:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSlotDomain">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblSlotVal">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Значение:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSlotDomain_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddSlot">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteSlot">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SlotEditorWnd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SlotEditorWnd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
